index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="index.css" />
+  <script defer  type="module" src="index.js"></script>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Awesome Books</title>
+</head>
+<body>
+  <header>
+    <div class="navbar">
+      <h1>Awesome books</h1>
+      <ul class="navbar_menu">
+        <li class="navbar_menu_item" id="navList">List</li>
+        <li class="navbar_menu_item" id="navAdd">Add new</li>
+        <li class="navbar_menu_item" id="navContact">contact</li>
+      </ul>
+    </div>
+    <div id="datetime"></div>
+  </header>
+  <main id="listSection">
+  <h2>All Awesome Books</h2>  
+  <div class="container">
+
+  </div>
+</main>
+  <main class="hidden" id="addSection">
+  <h2>Add a new book</h2>
+  <form id="form">
+    <input id="booksTitle" type="text" name="title" placeholder="Title">
+    <input id="booksAuthors" type="text" name="author" placeholder="Author">
+  </form>
+  <button class="addButton">Add</button>
```

Comment:
- Brilliant work :muscle: 
 - [ ] [OPTIONAL] For more user-friendliness, I think it would be nice if the form can be reset after submission of the book data without the user making the mistake of adding it again or having to erase all by themselves before they could add another :thinking: 
 
![image](https://github.com/Yulianav09/Awesome-books-ES6/assets/99715640/3f469158-8946-4f69-992a-ca6553ce3052)